backend/src/openarchiefbeheer/utils/tests/gherkin.py

```diff
@@ -75,44 +79,73 @@ class Given:
         def __init__(self, testcase):
             self.testcase = testcase
 
+        async def list_exists(self, **kwargs):
+            @sync_to_async()
+            def add_items(destruction_list, zaken):
+                for zaak in zaken:
+                    item = DestructionListItemFactory.create(
+                        destruction_list=destruction_list, zaak=zaak.url
+                    )
+                    destruction_list.items.add(item)
+
+            record_manager = await self.record_manager_exists()
+            base_kwargs = {
+                "name": "My First Destruction List",
+                "assignee": record_manager,
+                "author": record_manager,
+            }
+            merged_kwargs = {**base_kwargs, **kwargs}
+            destruction_list = await self._get_or_create(
+                DestructionListFactory, **merged_kwargs
+            )
+            zaken = await self.zaken_are_indexed(100)
+            await add_items(destruction_list, zaken)
+            return destruction_list
+
+        async def assignee_exists(self, **kwargs):
```

Comment:
This only enables to have the record manager as assignee :thinking:

Reply:
I think we can do:

```python
reviewer = await self.given.reviewer_exists()
await self.given.list_exists(assignee=reviewer)
```

Reply:
Ah ok!